06-Modelling-Rigid-Bodies/01-Introduction-to-Rigid-Bodies.ipynb

```diff
@@ -537,8 +537,6 @@
     "    \"constituent_types\": ['A', 'A'],\n",
```

Comment:
<p>Good idea! We should modify HOOMD to set the particle types as well. That will ensure that <code>requestExtraGhostLayerWidth</code> always produces correct results (e.g. in cases where users use the local access arrays to change their particle types).</p><p> </p><p>Currently, HOOMD does not set the particle types. It uses <code>constituent_types</code> in <code>create_bodes</code>, the validation step, and in <code>requestExtraGhostLayerWidth</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Good eye, I missed that we set `postype` to the current particle's type. I will make a PR with this change for the 4.0 release.